alpha-features.md

```diff
@@ -0,0 +1,28 @@
+# Open Service Broker API Alpha Features
+
+## Table of Contents
+  - [Alpha Overview](#overview)
+  - [How alpha features are documented](#how-alpha-features-are-documented)
+  - [Current alpha features](#current-alpha-features)
+
+## Overview
+
+The features described in this document are released at an ALPHA level.  They
+are are available for use in order for the community to provide feedback and
+are not yet an official part of the spec.
+
+Alpha features may be changed or removed in response to feedback.
+
+## How Alpha features are documented
+
```

Comment:
I'm interested in this also. If we agree that parameter schemas should be alpha before GA, then I recommend including the boilerplate for alpha features at the same time as adding parameter schemas to the spec. This will give us a concrete example to review legibility. 

Or provide a diff with a fork that mocks up the alpha boilerplate along with the parameter schemas spec changes so we can see it all in context?